benchmarks/ALIA/others/inc-array.sl

```diff
@@ -0,0 +1,17 @@
+(synth-inv inv_fun ((x (Array Int Int))))
```

Comment:
Missing `(set-logic ALIA)`?

I don't think you reran the array benchmarks after undoing the default logic change ... Please rerun them before pushing the commit